spec/std/socket/address_spec.cr

```diff
@@ -127,6 +127,16 @@ describe Socket::IPAddress do
     Socket::IPAddress.valid_port?(-1).should be_false
     Socket::IPAddress.valid_port?(65_536).should be_false
   end
+  it "#==" do
```

Comment:
Missing a newline before this

Reply:
Related https://github.com/crystal-lang/crystal/issues/9804.